build.sh

```diff
@@ -10,6 +10,34 @@ fi
 apk update
 apk add alpine-sdk util-linux strace file zlib-dev zlib-static autoconf automake libtool
 
+# Build static squashfuse
+apk add glib-dev  glib-static fuse-dev fuse-static zstd-dev zstd-static # fuse3-static fuse3-dev
+# wget -c -q "https://github.com/vasi/squashfuse/releases/download/0.1.104/squashfuse-0.1.104.tar.gz"
+wget -c -q "https://github.com/vasi/squashfuse/archive/refs/heads/master.zip"
```

Comment:
it's probably preferably to use a specific commit hash here rather than `master` in order to have a reproducible result and not breaking the build randomly at the whims of upstream changes.

Reply:
btw, you can just use .tar.gz instead of .zip on this url, so you can use standard unix tools. also zip format doesnt support unix permissions.

Reply:
Thanks @rofl0r. Did that.